test/unittest/participant/participant_factory/ParticipantFactoryTest.cpp

```diff
@@ -30,18 +31,51 @@ namespace eprosima {
 namespace ddsrouter {
 namespace test {
 
+// TODO: refactor these tests so they test with type in id and with type in Configuration
+// TODO: add these functions in a common utils test
+void set_domain(
+        RawConfiguration& configuration,
+        uint16_t seed = 0)
+{
+    configuration[DOMAIN_ID_TAG] = seed;
+}
+
+ParticipantConfiguration get_random_participant_configuration(
```

Comment:
I dont see anything random here and I would not randomize it as we always want to know every cases in the tests

```suggestion
ParticipantConfiguration get_participant_configuration(
```

Reply:
The proper name would be `arbitrary`. It does not mean to be random, but to create different random configurations depending on the seed, so we do not always use the main ones and forget about corner or complex cases.
We'll attempt this in future PRs.